src/diffusers/schedulers/scheduling_utils.py

```diff
@@ -41,3 +53,77 @@ class SchedulerMixin:
     """
 
     config_name = SCHEDULER_CONFIG_NAME
+
+    @classmethod
+    def from_pretrained(cls, config: Dict[str, Any] = None, return_unused_kwargs=False, **kwargs):
+        r"""
+        Instantiate a Scheduler class from a pre-defined JSON-file.
+
+        Parameters:
+            pretrained_model_name_or_path (`str` or `os.PathLike`, *optional*):
+                Can be either:
+
+                    - A string, the *model id* of a model repo on huggingface.co. Valid model ids should have an
+                      organization name, like `google/ddpm-celebahq-256`.
+                    - A path to a *directory* containing model weights saved using [`~ConfigMixin.save_config`], e.g.,
```

Comment:
```suggestion
                    - A path to a *directory* containing model weights and configurations saved using [`~ConfigMixin.save_config`], e.g.,
```

It also works for models and schedulers saved with `save_pretrained`, right?

Reply:
Ah yeah also a good point - should correct the docs here